src/main/java/gov/cabinetoffice/gap/spotlight/publisher/enums/ResponseTypeEnum.java

```diff
@@ -0,0 +1,29 @@
+package gov.cabinetoffice.gap.spotlight.publisher.enums;
+
+import java.util.Map;
+
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.LONG_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.NO_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.NUMERIC_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.SHORT_ANSWER_VALIDATION;
+import static gov.cabinetoffice.gap.spotlight.publisher.costants.ValidationMaps.SINGLE_FILE_UPLOAD_VALIDATION;
+
+
+public enum ResponseTypeEnum {
+
+    // TODO update validation on AddressInput and Date
+    YesNo(NO_VALIDATION), SingleSelection(NO_VALIDATION), Dropdown(NO_VALIDATION), MultipleSelection(NO_VALIDATION),
```

Comment:
Enum names should be uppercase e.g YES_NO, SINGLE_SELECTION etc

Reply:
Will have a look at this to confirm but I think these enum names are saved as part of the questionnaire builder and need to be like this. Suspect it would be significant effort to refactor across the full service.